src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/infrastructure/clients/model/idam/UserInfo.java

```diff
@@ -11,10 +11,10 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class UserInfo {
 
-    @JsonProperty("sub")
```

Comment:
this line is needed because Idam returns email in sub field. You may get error with real example

Reply:
Added back @JsonProperty("sub")